gatsby/content/faq/bridging.mdx

```diff
@@ -0,0 +1,16 @@
+---
+section: faq
+faq_section: bridging
+---
+
+#### Why don't you use separate homeserver instances to improve bridging performance?
+
+For example:
+
+> Is there any merit in a proposal that the current matrix.org Freenode bridge could be taken off matrix.org and put on a new HS (at a new server-name domain such as freenode.matrix.org) so as to not be bogged down in performance?
+
+In this case you are now maintaining two high traffic homeservers rather than one: Freenode handles enough traffic that it's intensive just by itself.
```

Comment:
Something about this statement feels wrong in tone. I know I wrote it, but maybe it needs a more positive tone somehow...

Something along the lines of "The matrix.org foundation currently runs one homeserver, matrix.org. This homeserver is operationally intensive and can be slow. However running two homeservers would not improve performance, because the work done to maintain one homeserver would need to be duplicated. Also, the traffic on the bridge instance would be intensive just by itself. Running two intensive homeservers, where optimisations such as caching and federation traffic batching would need to be duplicated would have an marked decrease in overall performance, to say nothing of the cost of running them."

Reply:
Maybe not verbatim, but maybe you can wordsmith it?

Reply:
Very good - added and edited.